frontend/packages/dev-console/integration-tests/support/pages/app.ts

```diff
@@ -32,8 +32,7 @@ export const perspective = {
     nav.sidenav.switcher.changePerspectiveTo(perspectiveName);
     app.waitForLoad();
     if (switchPerspective.Developer) {
-      // Bug: 1890676 is created related to Accessibility violation - Until bug fix, below line is commented to execute the scripts in CI
-      // cy.testA11y('Developer perspective with guider tour modal');
+      cy.testA11y('Developer perspective with guide tour modal');
```

Comment:
@jerolimov - If we uncomment this line, it stops all other scripts, because it executes on every test scenario. So I recommend not to uncomment this line until all ally fixes are done

![image](https://user-images.githubusercontent.com/61005404/121478656-e2395000-c9e6-11eb-9088-3e971c035a6d.png)

Reply:
Sorry, then we need to merge #9124 first. Thanks for verifying and sorry that this alone doesn't work.

I will ping you again after the other PR is merged.

Reply:
The other PR is merged and I extended this PR as well. Can you check the tests here again. Thank you.